packages/favorites-dialog/src/EnhancedToolbar.js

```diff
@@ -31,6 +32,13 @@ const toolbarStyles = () => ({
     },
 });
 
+const VisTypeFilterMenuItem = withStyles(toolbarStyles)(({ classes, type, icon = undefined, label = undefined }) => (
```

Comment:
Whats the difference between `x = undefined` and just leaving it as `x`? Wouldn't it fallback to `undefined` if not provided?